web/src/queries/dasd.ts

```diff
@@ -135,61 +138,28 @@ const useSelectedDASD = (): DASDDevice[] => {
   const { data } = useQuery(selectedDASDQuery());
 
   return data || [];
-}
+};
 
 const useSelectedDASDChange = () => {
   type SelectDASD = {
-    unselect?: boolean,
-    device?: DASDDevice,
-    devices?: DASDDevice[]
-  }
+    unselect?: boolean;
```

Comment:
NP: if I'm not wrong, there is somewhere an identical type called SelectionOptions.